workflow/scripts/copy_files.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import os
+import yaml
+
+
+def copy_files():
+    '''Copy all files relevant for the clinic to the output folder'''
+
+    # Get output folder using snakemake or, if running script independently, directly from config file
+    try:
```

Comment:
Hmm... I'm wondering if this should be moved to the if __name__ thing. There you could add the if-statement and also feed the function with the input. As it now stands I instinctively looked for `copy_files():` input.